roles/setup/tasks/setup-local.yml

```diff
@@ -0,0 +1,77 @@
+---
+# if bin not exists we download it from github repo/tag
+
+- name: Check in {{ local_receptor_path }}/receptor exist
+  ansible.builtin.stat:
+    path: "{{ local_receptor_path }}/receptor"
+  register: local_receptor_stat
+
+- name: Download receptor bin from github
+  block:
+    - name: Create local receptor path
+      ansible.builtin.file:
+        path: "{{ local_receptor_path }}"
+        state: directory
+        mode: '0644'
+
+    - name: Get latest release of receptor repository
+      ansible.builtin.uri:
+        url: "https://api.github.com/repos/{{ receptor_github_owner }}/\
+          {{ receptor_github_repo }}/releases/latest"
+      register: latest_receptor_release
+
+    - name: Set receptor downloads facts
+      ansible.builtin.set_fact:
+        receptor_tag: "{{ (receptor_github_release |
+          default(latest_receptor_release.json.tag_name)) }}"
+        receptor_arch: "{{ arch_mapping[ansible_architecture] | default(ansible_architecture) }}"
+
+    - name: Download receptor from {{ receptor_tag }} release
+      ansible.builtin.get_url:
+        url: "https://github.com/{{ receptor_github_owner }}/\
+          {{ receptor_github_repo }}/releases/\
+          download/{{ receptor_tag }}/{{ receptor_github_repo }}_\
+          {{ receptor_tag | regex_replace('^v', '') }}_{{ ansible_system | lower }}_{{ receptor_arch }}.tar.gz"
+        dest: "{{ local_receptor_path }}"
+        mode: '0644'
+
+    - name: Unarchive receptor release
+      ansible.builtin.unarchive:
+        src: "{{ local_receptor_path }}/{{ receptor_github_repo }}_\
+          {{ receptor_tag | regex_replace('^v', '') }}_{{ ansible_system | lower }}_{{ receptor_arch }}.tar.gz"
+        dest: "{{ receptor_install_dir }}"
+        remote_src: true
+
+    - name: Remove receptor archive
+      ansible.builtin.file:
+        path: "{{ local_receptor_path }}/{{ receptor_github_repo }}_\
+          {{ receptor_tag | regex_replace('^v', '') }}_{{ ansible_system | lower }}_{{ receptor_arch }}.tar.gz"
+        state: absent
+
+  when: not local_receptor_stat.stat.exists
+
+- name: Copy receptor's build to "{{ receptor_install_dir }}"
+  ansible.builtin.copy:
+    src: "{{ local_receptor_path }}/receptor"
+    dest: "{{ receptor_install_dir }}/receptor"
+    owner: root
+    group: root
+    mode: 'u=rwx,go=rx'
+    remote_src: true
+  when: local_receptor_stat.stat.exists
+
+- name: Add receptor's path to profile
+  ansible.builtin.copy:
+    dest: /etc/profile.d/receptor.sh
+    content: "PATH=$PATH:{{ receptor_install_dir }}/receptor"
+    owner: root
+    group: root
+    mode: 'u=rwx,go=rx'
+
+- name: Create receptor's systemd service
+  ansible.builtin.template:
+    src: templates/systemd_receptor.service.j2
+    dest: /lib/systemd/system/receptor.service
```

Comment:
later a configure.yml task will be ran, which expects to put an override systemd conf in a different directory

https://github.com/ansible/receptor-collection/blob/a9bc4d51cf438194f33f17c77fc12802d7bb46cb/roles/setup/tasks/configure.yml#LL21C19-L21C19

should these locations match?

Reply:
actually systemd seems to apply the override just fine
```bash
admin@ip-172-31-10-251:/lib/systemd/system$ systemctl status receptor
● receptor.service - Ansible Receptor for Linux
     Loaded: loaded (/lib/systemd/system/receptor.service; enabled; vendor preset: enabled)
    Drop-In: /etc/systemd/system/receptor.service.d
             └─override.conf
```